### PR TITLE
Completed all the tasks addressed in #9

### PR DESCRIPTION
**Pull request by diru1100**
Changed the issues in all HTML files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,28 +113,28 @@ <h2 class="section__header">Past Events</h2>
...
-            <img class="card__image" src="/img/events/AI-ML-session.jpg" loading="lazy">
+            <img class="card__image" src="/img/events/AI-ML-session.jpg" loading="lazy" alt="Artificial Intelligence and Machine Learning session">
             <div class="card__info">
               <p>AI/ML Session</p>
               <p>9 March 2019</p>
             </div>
           </div>
           <div class="event card">
-            <img class="card__image" src="/img/events/web-session.jpg" loading="lazy">
+            <img class="card__image" src="/img/events/web-session.jpg" loading="lazy" alt="Wbs Session for Juniors">
```
Comment (r0mflip) [inline index.html]: Typo in alt text

**Comment by r0mflip** [comment]
I think alt texts can be made a bit more descriptive. Can be a bit long, but helps if they are concise and a bit more contentful and meaningful.

Also do share screenshots/links of the results from lighthouse and webhint for the Netlify deployment.

**Comment by r0mflip** [comment]
https://github.com/cbitosc/cbitosc.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: index.html [hunk 2]
```diff
@@ -163,21 +163,22 @@ <h3 class="timeline-marker">2018</h3>
         <h3 class="timeline-marker">2017</h3>
         <div class="cards">
           <div class="event card">
-            <img class="card__image" src="/img/events/month-of-webvr.jpg" loading="lazy">
+            <img class="card__image" src="/img/events/month-of-webvr.jpg" loading="lazy" align="
```
Comment (r0mflip) [inline index.html]:
```suggestion
            <img class="card__image" src="/img/events/month-of-webvr.jpg" loading="lazy" alt="
```

Remove newline too.

File: index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
```
Comment (notlmn) [inline index.html]: Remove empty line here!

File: index.html [hunk 4]
```diff
@@ -145,16 +145,16 @@ <h3 class="timeline-marker">2019</h3>
         <h3 class="timeline-marker">2018</h3>
         <div class="cards">
           <div class="event card">
-            <img class="card__image" src="/img/events/hacktoberfest-2018.jpg" loading="lazy">
+            <img class="card__image" src="/img/events/hacktoberfest-2018.jpg" loading="lazy" alt="Hacktoberfest-2018 Poster">
```
Comment (notlmn) [inline index.html]: This is not a poster, refer to the source image for context.

**Comment by notlmn** [comment]
Closing due to inactivity, and the codebase has changed since then.

If a new PR if you have to.

**Pull request closed without merging**